readme.md

```diff
@@ -276,6 +277,35 @@ deno run --allow-read --allow-run \
   "some string to find" "replacement string" # optional
 ```
 
+## 'mermaid'
```

Comment:
These need to be backticks

<img width="301" alt="Screenshot 2023-05-27 at 8 08 16 PM" src="https://github.com/pskfyi/handy/assets/47710574/e4177695-aceb-44c4-9a3d-488bedaa7110">

Reply:
Thanks!